src/sql/src/query_model/mod.rs

```diff
@@ -149,7 +153,9 @@ pub enum BoxType {
 }
 
 #[derive(Debug)]
-pub struct BaseTable {/* @todo table metadata from the catalog */}
+pub struct Get {
+    id: expr::GlobalId,
```

Comment:
Eventually, we'll need to add the RelationType here too, when we implement the `typ` method in `BoxScalarExpr`.

Reply:
We could add it as a follow up task and add an option to the dot generator to dump types.

Reply:
Should types just included in the dot graph always? Or is the concern that if a column is something like record type, the record type can be very long?

Reply:
Added #9601 as a follow-up issue.